test/unit/openapi/streaming/subscription-spec.js

```diff
@@ -387,7 +387,10 @@ describe("openapi StreamingSubscription", () => {
 			});
 		});
 
-		it ("ignores a unsubscribe followed by a subscribe when waiting for an action to respond", (done) => {
+		/**
+		 * Unsubscribe before subscribe is required for modify action.
+ 		 */
+		it ("accept unsubscribe followed by a subscribe when waiting for an action to respond", (done) => {
```

Comment:
Did you mean unsubscribe followed by subscribe? Cause it's what you linked to.

Anyhow, digesting that idea. I know we talked about it in December/January. But I'm not sure if it's worth complicating subscription.